### ISSUE TITLE
Ticket orders cannot be created via the CP

### ISSUE THREAD

**Issue by mihob** [verbb/Events#165]
### Describe the bug

It is not possible to create an order and add line items via the CP.

The selection of line items remains empty although there should be available tickets. 

<img width="1064" alt="grafik" src="https://github.com/user-attachments/assets/a8432d40-8c0f-4838-825d-89ff0dd60a44">

The corresponding Ajax request fails with the following error message:

```
{
    "name": "Exception",
    "message": "Calling unknown method: verbb\\events\\elements\\LegacyTicket::getSalePrice()",
    "code": 0,
    "exception": "yii\\base\\UnknownMethodException",
    "file": "/app/vendor/yiisoft/yii2/base/Component.php",
    "line": 308,
    "trace": [
        {
            "file": "/app/vendor/craftcms/cms/src/base/Element.php",
            "line": 2537,
            "function": "__call",
            "class": "yii\\base\\Component",
            "type": "->"
        },
        {
            "file": "/app/vendor/craftcms/commerce/src/controllers/OrdersController.php",
            "line": 1867,
            "function": "__call",
            "class": "craft\\base\\Element",
            "type": "->"
        },
        {
            "file": "/app/vendor/craftcms/commerce/src/controllers/OrdersController.php",
            "line": 747,
            "function": "_addLivePurchasableInfo",
            "class": "craft\\commerce\\controllers\\OrdersController",
            "type": "->"
        },
        {
            "function": "actionPurchasablesTable",
            "class": "craft\\commerce\\controllers\\OrdersController",
            "type": "->"
        },
        {
            "file": "/app/vendor/yiisoft/yii2/base/InlineAction.php",
            "line": 57,
            "function": "call_user_func_array"
        },
        {
            "file": "/app/vendor/yiisoft/yii2/base/Controller.php",
            "line": 178,
            "function": "runWithParams",
            "class": "yii\\base\\InlineAction",
            "type": "->"
        },
        {
            "file": "/app/vendor/yiisoft/yii2/base/Module.php",
            "line": 552,
            "function": "runAction",
            "class": "yii\\base\\Controller",
            "type": "->"
        },
        {
            "file": "/app/vendor/craftcms/cms/src/web/Application.php",
            "line": 350,
            "function": "runAction",
            "class": "yii\\base\\Module",
            "type": "->"
        },
        {
            "file": "/app/vendor/craftcms/cms/src/web/Application.php",
            "line": 649,
            "function": "runAction",
            "class": "craft\\web\\Application",
            "type": "->"
        },
        {
            "file": "/app/vendor/craftcms/cms/src/web/Application.php",
            "line": 312,
            "function": "_processActionRequest",
            "class": "craft\\web\\Application",
            "type": "->"
        },
        {
            "file": "/app/vendor/yiisoft/yii2/base/Application.php",
            "line": 384,
            "function": "handleRequest",
            "class": "craft\\web\\Application",
            "type": "->"
        },
        {
            "file": "/app/web/index.php",
            "line": 13,
            "function": "run",
            "class": "yii\\base\\Application",
            "type": "->"
        }
    ]
}
```

### Steps to reproduce

1.  Create a new order via the CP
2. Try to add line items

### Craft CMS version

5.5.0.1

### Plugin version

3.0.1

### Multi-site?

YES

**Comment by engram-design** [comment]
Fixed for the next release. To get this early, run `composer require verbb/events:"dev-craft-5 as 3.0.2"`.

**Comment by engram-design** [comment]
Fixed in [3.0.3](https://github.com/verbb/events/releases/tag/3.0.3)
